src/elm/Page/PublicProfile.elm

```diff
@@ -0,0 +1,225 @@
+module Page.PublicProfile exposing (Model, Msg, init, jsAddressToMsg, msgToString, update, view)
+
+import Api.Graphql
+import Avatar
+import Eos
+import Eos.Account as Eos
+import Graphql.Http
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import I18Next exposing (t)
+import Icons
+import Json.Decode exposing (Value)
+import Page
+import Profile exposing (Profile)
+import Route
+import Session.LoggedIn as LoggedIn exposing (External(..), FeedbackStatus(..))
+import Session.Shared exposing (Shared)
+import UpdateResult as UR
+
+
+init : LoggedIn.Model -> String -> ( Model, Cmd Msg )
+init loggedIn accountName =
+    let
+        profileQuery =
+            Api.Graphql.query loggedIn.shared
+                (Profile.query (Eos.stringToName accountName))
+                CompletedProfileLoad
+    in
+    ( initModel loggedIn
+    , Cmd.batch
+        [ profileQuery
+        ]
+    )
+
+
+type alias UpdateResult =
+    UR.UpdateResult Model Msg (External Msg)
+
+
+type Msg
+    = CompletedProfileLoad (Result (Graphql.Http.Error (Maybe Profile)) (Maybe Profile))
+
+
+type alias Model =
+    { status : Status
+    }
```

Comment:
Model can be `Status` directly, using records with only one field generate a `elm-analyse` warning